src/main.rs

```diff
@@ -545,18 +598,17 @@ fn run() -> Result<()> {
                 })
                 .unwrap_or_else(|| vec![None]); // read from stdin (None) if no args are given
 
+            let output_components = values_t!(app_matches.values_of("style"), OutputComponent)?
+                .into_iter()
+                .map(|style| style.components())
+                .fold(vec![], |mut acc, x| {
```

Comment:
`Vec::new()` would be better since it's empty right?

Reply:
`vec![]` is empty, too. I guess this is just preference, but results in the same code. ([1](https://github.com/rust-lang-nursery/rust-clippy/issues/1542), [2](https://github.com/rust-lang-nursery/fmt-rfcs/issues/70))